### ISSUE TITLE
mtl blue and red: Populate all guides and resources at mtl.how

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#830]
Need to carefully move things from https://github.com/lzim/mtl/blob/lzim-patch-2/ to the main mtl repo, which will be https://mtl.how

Stacey's Next Step:
- [x] 1. Upload the mtl_1.7 and mtl_1.8 models to the lzim-patch-2 Blue folder
- [x] 2. Add "blue" for the Blue folder in the file path for all the current mtl.how content at lzim-patch-2
- [x] 3. Re-upload the resource files into the new Blue folder
- [x] 4. Add the session guides for the Red folder

_Add estimated person-hours to complete:_
Four hours for moving.
Perhaps more on Red guides.

**DUE DATE**
- Before local team meetings: January 7, 2020

**Comment by staceypark** [comment]
- [ ] mtl 1.8 models needs a dummy file like we have in mtl 1.7 models

**Comment by lzim** [comment]
@staceypark @TomRust 

Let’s create this file using Nevermore. Tom can help make sure it still works with SP model 1.8.

**Comment by branscombj** [comment]
@staceypark @lzim @dlkibbe - 2 dependency questions

1. I think - please confirm - that I need to wait for all the learner/facilitator resources to be placed into the new, split, mtl.how repo before I can do my task of correcting all the icon links (so they link to the page you're on and neither throw a 401 error nor link to the image file in our TeamPSD repo)?

2. I think that we want to get MTL Red materials to Elizabeth soon, right? The revised brochures and See guides are in #855  and the remaining EES materials - Say guides and post tests - are pending any final edits/approvals on those. (@dlkibbe is working on post-tests based on the See guides as proposed; don't want to get too far down the road on that and Say if the Sees are not in line.

**Comment by dlkibbe** [comment]
@staceypark @lzim @branscombj 
I have drafted 12 questions (I know only 10 is needed) for Red post-test 1 based on See guide. I want to go on to data UI to make sure all phrasing is correct before I share with all of you.

**Comment by branscombj** [comment]
@staceypark Please see issue #858

**Comment by staceypark** [comment]
@branscombj the guides are all already split on the mtl.how repo into the red and blue folders.
As you noted yesterday however, there are several facilitator guides missing in the earlier session folders.

Since you need to have links to do the task of updating session guide image links to point back to the session guide, you could use another link and update the session number instead:
i.e. Take the session 06 say guide link:
https://github.com/lzim/mtl/blob/master/blue/session06/s06_facilitator/mtl_session06_say.md

And change the "06" numbers throughout to say "01". Since _all folders are built the same way_, this would point to the Session 01 Say guide that we would post:
https://github.com/lzim/mtl/blob/master/blue/session01/s01_facilitator/mtl_session01_say.md

As @lzim details in #601, I believe the guides at the **teampsd repo** will be the most up to date. You will need to 
1) start with the guides at the **teampsd repo**
2) update the links making sure to **add** (instead of replace) your initials and date in the "date" section of the YAML header (again see #601 for the decision on this)
3) create a pull request to @staceypark in the **teampsd repo** for approval
4) once approved in the **teampsd repo** move both the **SAY and SEE** guides to the **mtl.how repo** by copy + pasting all 12 SEE and SAY session guides
5) accept your own pull request at the **mtl.how repo**

FYI: @dlkibbe 

> @staceypark @lzim @dlkibbe - 2 dependency questions
> 
> 1. I think - please confirm - that I need to wait for all the learner/facilitator resources to be placed into the new, split, mtl.how repo before I can do my task of correcting all the icon links (so they link to the page you're on and neither throw a 401 error nor link to the image file in our TeamPSD repo)?
> 2. I think that we want to get MTL Red materials to Elizabeth soon, right? The revised brochures and See guides are in #855  and the remaining EES materials - Say guides and post tests - are pending any final edits/approvals on those. (@dlkibbe is working on post-tests based on the See guides as proposed; don't want to get too far down the road on that and Say if the Sees are not in line.

**Comment by staceypark** [comment]
@lzim Do you want @branscombj and @dlkibbe to also take a stab at the missing SAY checklists we still need to create?

**Comment by branscombj** [comment]
@staceypark I think I've got it (your comment 33 minutes ago). Thanks!

**Comment by staceypark** [comment]
@anthonycpichardo can you create a model parameters file similar to how we did for James but with all modules? We have post "dummy" params files on our mtl.how repository for folks who want to download the models and parameters data and play with it themselves.

@lzim can help you with clinic selections needed to create it

**Comment by anthonycpichardo** [comment]
Yeah I can do that when I have the clinic selection list.

@staceypark @lzim

**Comment by lzim** [comment]
> @lzim can help you with clinic selections needed to create it

@staceypark What do you mean?

Thanks 😆

**Comment by staceypark** [comment]
@anthonycpichardo @lzim 
We need to generate a dummy team data table file for MTL 1.8 & 2.0.
Do create this file with useful numbers, Anthony needs to know what set of clinics to choose to generate this data

**Comment by staceypark** [comment]
We decided last week that we would have MTL 1.7 and 2.0.

For the MTL 2.0 models and model parameters:
1. Tom is updating the SP model (tbd Monday) #648 
2. We need to generate a model parameters file that people can download with the MTL 2.0 files (See how we did MTL 1.7 here - https://github.com/lzim/mtl/tree/master/blue/mtl1.7_models). To create that model parameters file, we need a set of clinics.
